NDB.Covid19/NDB.Covid19/Models/PersonalDataModel.cs

```diff
@@ -32,7 +32,7 @@ public class PersonalDataModel
         [JsonIgnore]
         public bool UnknownStatus => Covid19_status == "ukendt";
         [JsonIgnore]
-        public bool AnonymousTokensEnabled => Covid19_anonymous_token == "available";
+        public bool AnonymousTokensEnabled => Covid19_anonymous_token == "v1:enabled";
```

Comment:
Did you test it locally?

Reply:
I have not tested the flow with the changes by running the actual flow, only checking that existing unit tests are running, and creating some ad-hoc unit tests, as I do not have the necessary tools/devices to build and deploy to an iPhone/Android.

I just added some temp unit tests to test this specific part by copying (most) of the logic in AuthenticationManager.GetPayloadValidateJWTToken, skipping the token signature validation. However this test as written will only work for the valid period of the JWT token, and also depends on copying code, rather than testing the actual implementation. Introducing test that would work over time, and test the actual implementation, would require some refactoring which I think is not appropriate for this PR.

Sample ad-hoc tests below:

```c#
[Fact]
public void TmpTokenParsingTest_AtEnabledInToken_AtEnabledShouldBeTrue()
{
    var accessTokenWithEnabledAt =
        "eyJhbGciOiJSUzI1NiIsImtpZCI6IkY5NjU2QzRCRkMwNDU4NkRDODQ0RDQyMzE0OEY2NTUwODgxNjgxMDkiLCJ0eXAiOiJhdCtqd3QiLCJ4NXQiOiItV1ZzU193RVdHM0lSTlFqRkk5bFVJZ1dnUWsifQ.eyJuYmYiOjE2MTE4MzIzOTUsImV4cCI6MTYxMTgzNTk5NSwiaXNzIjoiaHR0cHM6Ly9sb2NhbGhvc3Q6NTAwMSIsImF1ZCI6Imh0dHBzOi8vbG9jYWxob3N0OjUwMDEvcmVzb3VyY2VzIiwiY2xpZW50X2lkIjoidGVzdC1zcGEtY2xpZW50Iiwic3ViIjoiZTQxYjBlYzQtMmJmYi00Nzc0LThiMzMtNmEyNjdmYzU4MGExIiwiYXV0aF90aW1lIjoxNjExODMyMzkzLCJpZHAiOiJpZHBvcnRlbiIsImNvdmlkMTlfc3RhdHVzIjoicG9zaXRpdiIsImNvdmlkMTlfc21pdHRlX3N0YXJ0IjoiMjAyMS0wMS0yMCIsImNvdmlkMTlfYmxva2VyZXQiOiJmYWxzZSIsInJvbGUiOiJ1cGxvYWQtYXBwcm92ZWQiLCJjb3ZpZDE5X2Fub255bW91c190b2tlbiI6InYxOmVuYWJsZWQiLCJqdGkiOiI1NDJFMUUzNEE5MTQ5NEY4NUIzNjYwQzg2MEY0QzNBQyIsInNpZCI6IkFCQ0Q5QzlEOURFMDg3RDIwQzE2ODcwRDNCQjY3NzY1IiwiaWF0IjoxNjExODMyMzk1LCJzY29wZSI6WyJvcGVuaWQiLCJzbWl0dGVzdG9wIl0sImFtciI6WyJleHRlcm5hbCJdfQ.jjKPqT5Tqfygo_Tp35hdigGok2qBVPJojCsTqr6ZoVeo1ppJEBLKp0dKBdKMwKLKMbqfvk0sR22zPkCrBI26eCJRH-aw4TbbKdgwCFzKRJajO0NGLGZHFSSadWIcRcO5_hYa-GZGqN8SbJS-spIuY9vYSLgFNkjj5Ua6b7zmal98d0CKd_RzTkxQ7EEwr6vNTAExtMVDwnWWCio6hqpXFXCCuOaNIraTDks81ORchiCzQ9rAWJVaTWTX0xGHLNLKEM-OdVfVOshTYpegrY6p-WWIvfd5e-WDM3zHSDkWd8LUSp_65MF880oq2QLLU7ikLcEuD846kDP32pn0Xb0E4A";
    var jsonSerialized = new JsonSerializer();
    string jsonPayload = new JwtBuilder()
        .WithValidator(new JwtValidator(new JsonNetSerializer(), new UtcDateTimeProvider(), 5 * 60))
        .Decode(accessTokenWithEnabledAt);

    System.Diagnostics.Debug.Print(jsonPayload);

    JObject obj = JObject.Parse(jsonPayload);

    PersonalDataModel personalDataModel = new PersonalDataModel();
    if (obj != null)
    {
        personalDataModel = obj.ToObject<PersonalDataModel>(jsonSerialized);
    }

    personalDataModel.AnonymousTokensEnabled.Should().BeTrue();
}

[Fact]
public void TmpTokenParsingTest_AtDisabledInToken_AtEnabledShouldBeFalse()
{
    var accessTokenWithDisabledAt =
        "eyJhbGciOiJSUzI1NiIsImtpZCI6IkY5NjU2QzRCRkMwNDU4NkRDODQ0RDQyMzE0OEY2NTUwODgxNjgxMDkiLCJ0eXAiOiJhdCtqd3QiLCJ4NXQiOiItV1ZzU193RVdHM0lSTlFqRkk5bFVJZ1dnUWsifQ.eyJuYmYiOjE2MTE4MzI0NzgsImV4cCI6MTYxMTgzNjA3OCwiaXNzIjoiaHR0cHM6Ly9sb2NhbGhvc3Q6NTAwMSIsImF1ZCI6Imh0dHBzOi8vbG9jYWxob3N0OjUwMDEvcmVzb3VyY2VzIiwiY2xpZW50X2lkIjoidGVzdC1zcGEtY2xpZW50Iiwic3ViIjoiNWQ1ZDIyZjEtMmMzMS00M2M4LTg1YTAtOTBhMjQ1NjA4YmNhIiwiYXV0aF90aW1lIjoxNjExODMyNDc3LCJpZHAiOiJpZHBvcnRlbiIsImNvdmlkMTlfc3RhdHVzIjoicG9zaXRpdiIsImNvdmlkMTlfc21pdHRlX3N0YXJ0IjoiMjAyMS0wMS0yMCIsImNvdmlkMTlfYmxva2VyZXQiOiJ0cnVlIiwiY292aWQxOV9saW1pdF9kdXJhdGlvbiI6IjI0IiwiY292aWQxOV9saW1pdF9jb3VudCI6IjMiLCJqdGkiOiIzODlBQzQzOEM0Q0EwNDIwQzU0QzE2QzVENzA3NDQzQyIsInNpZCI6IjlDOTU0RDc0QzlCOTY2NjRDOUVBQTAxNURFNEYxMUNGIiwiaWF0IjoxNjExODMyNDc4LCJzY29wZSI6WyJvcGVuaWQiLCJzbWl0dGVzdG9wIl0sImFtciI6WyJleHRlcm5hbCJdfQ.aTA-bLy9oKf7Pr_4ZcA5IOl4vjVohMEqsXeoLIXmYRBHB0-sshcON_ZA_kTKfniYda2DA2JvO4XWChx8hRTRVno3JcXWhz2vCIHYyi2LHj5xd9jq2WyxNeCdWEzuckwsgl5J3sjIaWusGVWiHX2GpGA4woWLQ1KcsKvQHqW2jQhtF2f8WzYNBJOdqdTUdP9jTMnAtqSf1CCNic2tdBNqSWi_9pF1mq3EBTPGSFci3XY4B_c6oPa2saMzBRQtqb0tN2Y2-am2divA1Otov9kvyJodkvdFQmksxEVpBs2EmdFgoF_JRe0RetGmo7b2V5ah4RuJSL1QgBqG4dj4EeQ9JA";
    var jsonSerialized = new JsonSerializer();
    string jsonPayload = new JwtBuilder()
        .WithValidator(new JwtValidator(new JsonNetSerializer(), new UtcDateTimeProvider(), 5 * 60))
        .Decode(accessTokenWithDisabledAt);

    System.Diagnostics.Debug.Print(jsonPayload);

    JObject obj = JObject.Parse(jsonPayload);

    PersonalDataModel personalDataModel = new PersonalDataModel();
    if (obj != null)
    {
        personalDataModel = obj.ToObject<PersonalDataModel>(jsonSerialized);
    }

    personalDataModel.AnonymousTokensEnabled.Should().BeFalse();
}
```

I'll hold off on merging until you have approved my test method :)

Reply:
Can we replace **:** with **_** in v1:enabled?

Reply:
Sure, done!